ballerina/Dependencies.toml

```diff
@@ -87,7 +87,7 @@ dependencies = [
 [[package]]
 org = "ballerina"
 name = "io"
-version = "1.2.0"
+version = "1.2.1"
```

Comment:
@dilanSachi Do we need to update this?

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-kafka/pull/427